src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/extension.ts

```diff
@@ -119,39 +125,56 @@ export async function activate(context: ExtensionContext, languageServerDir: str
                 documentManager,
                 languageServiceClient,
                 logger);
+            const semanticTokenProvider = new RazorDocumentSemanticTokensProvider(
+                documentSynchronizer,
+                documentManager,
+                languageServiceClient,
+                logger);
+
+            const tokenTypes = [
```

Comment:
Why do you need to specify this? I imagine this would have been handled on the language server front since you have an endpoint that exposes the legend/types

Reply:
You're right, I just failed to update it.